demos/README.md

```diff
@@ -2,3 +2,11 @@
 
 This directory contains applications that demonstrate how to use ExoPlayer.
 Browse the individual demos and their READMEs to learn more.
+
+To run a demo:
+
+- Open Android Studio
+- File -> New -> Import Project -> Specify the root ExoPlayer folder
+- Choose the demo from the run configuration dropdown list
+- Click Run
```

Comment:
Do you think it is worth giving the gradle instruction too?

Reply:
Good point. Will add.